packages/host/tests/helpers/mock-matrix-service.ts

```diff
@@ -282,6 +283,19 @@ function generateMockMatrixService(
         room.created?.getTime()
       );
     }
+
+    @cached
+    get roomResources() {
+      let resources: Map<string, RoomResource> = new Map();
+      for (let roomId of this.rooms.keys()) {
+        resources.set(
+          roomId,
+          getRoom(this, () => roomId),
+        );
+      }
+
+      return resources;
+    }
```

Comment:
same as previous comment--lets not create resources inside of getters. if we need a getter with side effects, let's fully document why this is necessary in the comments